packages/react-charting/src/components/TreeChart/TreeChart.styles.ts

```diff
@@ -18,7 +18,7 @@ export const getStyles = (props: ITreeStyleProps): ITreeStyles => {
     rectNode: {
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31439&commitId=65cebf86ca2fbb74d2b6ec64107d2bbcd51fdc25&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
Tree chart changes are reverted. 
Pivot changes are unrelated.